### PR TITLE
os/bluestore: batch throttle

### PR DESCRIPTION
**Pull request by majianpeng**
1:make throttle.put batch handle 
2: avoid call the same osd qcond.notify_all many times.

### CODE REVIEW

File: src/os/bluestore/BlueStore.h [hunk 1]
```diff
@@ -1863,6 +1863,9 @@ class BlueStore : public ObjectStore,
 
   std::atomic<uint64_t> max_blob_size = {0};  ///< maximum blob size
 
+  std::atomic<uint64_t> total_ios = {0};
```
Comment (ifed01) [inline src/os/bluestore/BlueStore.h]: no need in atomics here - they are accessed under kv_lock only

Comment (liewegas) [inline src/os/bluestore/BlueStore.h]: i agree we should make this non-atomic.  also define it next to the kv_* stuff and comment that it is defined as the sum of ios and costs for kv_queue.  Perhaps these should be name kv_queue_ios and kv_queue_cost

**Comment by liewegas** [comment]
the rest looks good!

**Comment by majianpeng** [comment]
update.

**Comment by liewegas** [comment]
http://pulpito.ceph.com/sage-2017-05-27_01:05:11-rados-wip-sage-testing---basic-smithi/

**Pull request merged**